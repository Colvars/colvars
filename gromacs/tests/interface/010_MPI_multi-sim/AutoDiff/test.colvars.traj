# step         d_atoms                coords_OW_337                                                              E_harmonic1          
           0    7.12545268541175e-01  (  2.71299067428490e+00 ,  2.22698516109300e+00 ,  2.83001964589859e+00 )   3.19499441701454e+03
           1    7.18778915858948e-01  (  2.71049102072852e+00 ,  2.23155319962084e+00 ,  2.83176218637769e+00 )   3.17274855058862e+03
           2    7.26705599877252e-01  (  2.70859342353356e+00 ,  2.23584090396576e+00 ,  2.83421366095066e+00 )   3.14457302950670e+03
           3    7.36198180861579e-01  (  2.70718214849530e+00 ,  2.23978214611882e+00 ,  2.83732845619262e+00 )   3.11099685719600e+03
           4    7.47063650144618e-01  (  2.70617604963423e+00 ,  2.24335612934103e+00 ,  2.84102007900367e+00 )   3.07278584664431e+03
           5    7.59135737117819e-01  (  2.70555891659587e+00 ,  2.24658937122445e+00 ,  2.84519201823374e+00 )   3.03060838178032e+03
           6    7.72359396181004e-01  (  2.70537905349941e+00 ,  2.24953991305581e+00 ,  2.84977072252406e+00 )   2.98474205596407e+03
           7    7.86797671197755e-01  (  2.70571898858041e+00 ,  2.25227008651108e+00 ,  2.85472123370685e+00 )   2.93506221941343e+03
           8    8.02555754498784e-01  (  2.70664689444200e+00 ,  2.25481887509599e+00 ,  2.86003673947791e+00 )   2.88131696658519e+03
           9    8.19682369419815e-01  (  2.70817383057937e+00 ,  2.25719537678851e+00 ,  2.86570994574179e+00 )   2.82346733963861e+03
          10    8.38122743435259e-01  (  2.71024908358954e+00 ,  2.25940214454814e+00 ,  2.87171320859538e+00 )   2.76183601588715e+03
          11    9.98875354054461e-01  (  3.01731065320951e+00 ,  2.08468909115544e+00 ,  2.75100344501888e+00 )   2.25337520266512e+03
          12    9.94574309275095e-01  (  3.01546390061308e+00 ,  2.08519624175351e+00 ,  2.75185493174647e+00 )   2.26630651029456e+03
          13    9.91648493577185e-01  (  3.01426714271342e+00 ,  2.08561119216056e+00 ,  2.75293959923641e+00 )   2.27512426692797e+03
          14    9.90117997959865e-01  (  3.01372533130951e+00 ,  2.08597660815349e+00 ,  2.75422476461833e+00 )   2.27974366008473e+03
          15    9.90008750094072e-01  (  3.01384993563361e+00 ,  2.08637569989015e+00 ,  2.75565211340529e+00 )   2.28007357479247e+03
          16    9.91347314358363e-01  (  3.01465592745715e+00 ,  2.08688413112379e+00 ,  2.75717037135677e+00 )   2.27603292589372e+03
          17    9.94148922640459e-01  (  3.01615612603881e+00 ,  2.08752108135997e+00 ,  2.75876727561743e+00 )   2.26758746718489e+03
          18    9.98402398774388e-01  (  3.01835727386790e+00 ,  2.08823891868560e+00 ,  2.76047640135037e+00 )   2.25479535600651e+03
          19    1.00406253702204e+00  (  3.02125927428157e+00 ,  2.08896028721472e+00 ,  2.76235257332762e+00 )   2.23782889314095e+03
          20    1.01105966795736e+00  (  3.02485631185728e+00 ,  2.08962977203884e+00 ,  2.76443495602414e+00 )   2.21694331238326e+03
          21    1.01932609960967e+00  (  3.02914019404755e+00 ,  2.09024532393872e+00 ,  2.76672209588959e+00 )   2.19239519929713e+03
          22    1.02882713071456e+00  (  3.03410641290257e+00 ,  2.09085952866036e+00 ,  2.76917003537638e+00 )   2.16434961132154e+03
          23    1.03957472482596e+00  (  3.03975755913956e+00 ,  2.09155113507388e+00 ,  2.77171455547126e+00 )   2.13284198436717e+03
          24    1.05160224766681e+00  (  3.04609422665050e+00 ,  2.09237543270775e+00 ,  2.77430876672529e+00 )   2.09785604896385e+03
          25    1.06490708756016e+00  (  3.05309313054414e+00 ,  2.09332630811444e+00 ,  2.77695101510509e+00 )   2.05949166733506e+03
          26    1.07940847418257e+00  (  3.06069171067991e+00 ,  2.09434390822954e+00 ,  2.77967874574021e+00 )   2.01808028322430e+03
          27    1.09496384703184e+00  (  3.06879967560020e+00 ,  2.09536108759795e+00 ,  2.78253309999908e+00 )   1.97412659114758e+03
          28    1.11143250023791e+00  (  3.07733224520463e+00 ,  2.09634942077878e+00 ,  2.78552461372984e+00 )   1.92811970139555e+03
          29    1.12873821000223e+00  (  3.08624204648343e+00 ,  2.09733843755996e+00 ,  2.78862132315921e+00 )   1.88035889670789e+03
          30    1.14689471374037e+00  (  3.09553263373381e+00 ,  2.09840713812458e+00 ,  2.79175807275358e+00 )   1.83089391570377e+03
          31    9.70298126307271e-01  (  3.01464505583404e+00 ,  2.08271950327462e+00 ,  2.75741890356583e+00 )   2.33998782237905e+03
          32    9.66732106284174e-01  (  3.01366375047440e+00 ,  2.08221046145304e+00 ,  2.75810427356094e+00 )   2.35091043389976e+03
          33    9.64698894035587e-01  (  3.01342876498085e+00 ,  2.08169128402006e+00 ,  2.75886236981794e+00 )   2.35714948597555e+03
          34    9.64142642420610e-01  (  3.01390028291449e+00 ,  2.08110794531761e+00 ,  2.75976767393987e+00 )   2.35885782283075e+03
          35    9.64996698383417e-01  (  3.01501951575945e+00 ,  2.08041698560150e+00 ,  2.76090452623855e+00 )   2.35623513597381e+03
          36    9.67229065311067e-01  (  3.01673868447276e+00 ,  2.07962047320563e+00 ,  2.76231834158917e+00 )   2.34938673822719e+03
          37    9.70863893056182e-01  (  3.01904666558850e+00 ,  2.07876887554744e+00 ,  2.76399139343555e+00 )   2.33825723355929e+03
          38    9.75977363215953e-01  (  3.02197895667742e+00 ,  2.07794121143799e+00 ,  2.76584966686949e+00 )   2.32264499743020e+03
          39    9.82660682570641e-01  (  3.02559923828419e+00 ,  2.07721172430661e+00 ,  2.76780355391083e+00 )   2.30231860421699e+03
          40    9.90952382910204e-01  (  3.02994911518648e+00 ,  2.07661779188287e+00 ,  2.76980591248238e+00 )   2.27722471064439e+03
          41    1.00078333733160e+00  (  3.03500032146178e+00 ,  2.07614973225037e+00 ,  2.77187986776137e+00 )   2.24765060162258e+03
          42    1.01198683683250e+00  (  3.04065695120518e+00 ,  2.07576556975924e+00 ,  2.77408926736643e+00 )   2.21418317375976e+03
          43    1.02437249063708e+00  (  3.04680909791810e+00 ,  2.07541555527632e+00 ,  2.77648282952048e+00 )   2.17747654638860e+03
          44    1.03781083275490e+00  (  3.05339246885731e+00 ,  2.07506362813623e+00 ,  2.77906109781834e+00 )   2.13799716080892e+03
          45    1.05228106564802e+00  (  3.06041751970348e+00 ,  2.07470249225757e+00 ,  2.78178118556649e+00 )   2.09589011288123e+03
          46    1.06785863004321e+00  (  3.06795806577202e+00 ,  2.07435847486790e+00 ,  2.78458508935245e+00 )   2.05102890354170e+03
          47    1.08463635337499e+00  (  3.07609867194605e+00 ,  2.07407672548285e+00 ,  2.78743248637634e+00 )   2.00325425218765e+03
          48    1.10262147637055e+00  (  3.08486550944088e+00 ,  2.07388873599498e+00 ,  2.79031660869558e+00 )   1.95266673830081e+03
          49    1.12169320052059e+00  (  3.09419373272120e+00 ,  2.07378696987677e+00 ,  2.79325131035640e+00 )   1.89972963349117e+03
          50    1.14165872380070e+00  (  3.10395934768268e+00 ,  2.07373093342659e+00 ,  2.79624285329490e+00 )   1.84509102262675e+03
