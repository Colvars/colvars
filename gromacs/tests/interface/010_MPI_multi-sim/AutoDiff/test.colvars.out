colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Start colvars Initialization.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "gromacs"
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "d_atoms"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = on
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 15.9994, total charge = -0.834.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 15.9994, total charge = -0.834.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "coords_OW_337"
colvars:   Initializing a new "cartesian" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 1 atoms requested: total mass = 15.9994, total charge = -0.834.
colvars:     # useX = on [default]
colvars:     # useY = on [default]
colvars:     # useZ = on [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { d_atoms }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 2.5 }
colvars:   # targetCenters = { 2.5 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 2000
colvars:   # decoupling = off [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "d_atoms" will be rescaled to 2000 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Current simulation parameters: initial step = 0, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "d_atoms":0/0. 1 atoms: total mass = 15.9994, total charge = -0.834.
colvars: Re-initialized atom group for variable "d_atoms":0/1. 1 atoms: total mass = 15.9994, total charge = -0.834.
colvars: Re-initialized atom group for variable "coords_OW_337":0/0. 1 atoms: total mass = 15.9994, total charge = -0.834.
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - GROMACS engine:
colvars:   Abraham2015 https://doi.org/10.1016/j.softx.2015.06.001
colvars: 
colvars: - Colvars module:
colvars: - Harmonic colvar bias implementation:
colvars: - Optimal rotation via flexible fitting:
colvars: - cartesian colvar component:
colvars: - distance colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - Colvars-GROMACS interface:
colvars:   n/a 
colvars: 
colvars: Setting initial step number from MD engine: 0
colvars: The final output state file will be "test.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: End colvars Initialization.
colvars: 
colvars: Saving collective variables state to "test.colvars.state".
